src/solvers/solver_utils.hpp

```diff
@@ -135,6 +145,155 @@ struct Stencil {
   }
 };
 
+namespace utils {
+template <class in, class out, bool only_fine_on_composite = true>
+TaskStatus CopyData(const std::shared_ptr<MeshData<Real>> &md) {
```

Comment:
This seems more generic than just solvers. Maybe deserves a space in some kind of just... `var` or `grid` utils...

Reply:
Maybe so, but I think the `only_fine_on_composite` stuff maybe makes it confusing for general usage? Let's see what others think.

Reply:
Perhaps just documenting `only_fine_on_composite` in a Doxygen comment could be the solution 😉

Reply:
ha, maybe so